src/countries/index.js

```diff
@@ -0,0 +1,15 @@
+import BRA from './countries/bra';
+import USA from './countries/usa';
+
+const countryObjectReferences = {
+  bra: BRA,
```

Comment:
Será que é valido trocar o nome das classes para o padrao ISO? Duvida real mesmo... To pensando se a gente n perde com legibilidade pra manter o codigo

Reply:
@guilhermeferrari  Entendo, foi uma forma que eu encontrei e que fez sentido para mim de escrever essa espécie de proxy, ele agiliza algumas coisas como a automação  sobre a inclusão de novos países, me baseei em um [código bem interessante em node](https://github.com/commenthol/date-holidays) e eles padronizam isso usando uma versão ainda mais enxuta do ISO, com apenas dois caracteres para representar, ex `BR`. Em questão de legibilidade, não acredito que interfira. E no caso de abrir para pessoas contribuírem, criamos um arquivo chamado `CONTRIBUTING.md`, e nesse arquivo específicamos como funciona e os passos básicos para inserir um novo país. A idéia desse modo que organizei é em suma evitar que precisemos mecher em vários arquivos do code base ao suportar um novo país. Ficou claro?

Reply:
Mas podemos discutir isso melhor :)